access.go

```diff
@@ -648,23 +664,21 @@ func (s *Server) FinishAccessRequest(w *Response, r *http.Request, ar *AccessReq
 		w.Output["expires_in"] = ret.ExpiresIn
 		if ret.RefreshToken != "" {
 			w.Output["refresh_token"] = ret.RefreshToken
-
-			refreshTokenJwt := decodeToken(ret.RefreshToken)
-			if refreshTokenJwt != nil {
-				w.Output["refresh_expires_in"] = refreshTokenJwt.Expiration - refreshTokenJwt.IssueAt
-				if !ar.SkipSetCookie {
-					AddTokenInCookie(w, ret.RefreshToken, "refresh_token", refreshTokenJwt.Expiration)
-				}
+			if ar.ExtendRefreshExpiration == true {
```

Comment:
for the extendRefreshExpiration, actually, I can't get the point here. whether extend the refresh, the logic should lie in the consumer side, not the token generation process. And also we should not pass the ExtendRefreshExpiration to AccessRequest. Is there anything I missing?

Reply:
ups yes... I think I miss understand about extendRefreshExpiration handling. I am going to do some adjustment today @angelo-chan

Reply:
Hi @angelo-chan and @marselsampe, do you have any suggestion or comment regarding this PR?